spec/src/main/doc/core/scopescontexts.asciidoc

```diff
@@ -437,116 +415,27 @@ Portable extensions are encouraged to fire an event with qualifier +@Initialized
 
 ==== Request context lifecycle
 
-The _request context_ is provided by a built-in context object for the built-in scope type +@RequestScoped+. The request scope is active:
-
-* during the +service()+ method of any servlet in the web application, during the +doFilter()+ method of any servlet filter and when the container calls any +ServletRequestListener+ or +AsyncListener+,
-* during any Java EE web service invocation,
-* during any remote method invocation of any EJB, during any asynchronous method invocation of any EJB, during any call to an EJB timeout method and during message delivery to any EJB message-driven bean, and
-* during +@PostConstruct+ callback of any bean.
```

Comment:
Is the "request scope available during @PostConstruct" feature intentionally restricted to Java EE only or is this an oversight?